### DISCUSSION TITLE
Difference between Base Prompt vs. Common Prompt?

### DISCUSSION THREAD

**Discussion by Eleksar387** [hako-mikan/sd-webui-regional-prompter#213] [Q&A]
I am struggling with the difference between base prompt and common prompt. Can someone provide examples when to use what?

**Comment by DylanComas** [comment]
If you've figured it out, I would also like to know.

**Comment by blugail** [comment]
Base prompts are mostly there for compatibility with "latent couple"
A common prompt adds the terms to each prompt (at the beginning I think)
A base prompt is a separate prompt that's run over the entire area in proportion to the "base ratio", so you might do something like:
Common prompt: Two girls sitting on a couch
Base prompt: A green tweed couch
Prompt one:Emma Watson
Prompt two:Emma Stone

The base prompt will keep the couch more green and tweed, but it will also weaken the generation of each character.
